projects/graph/src/graph_demo.py

```diff
@@ -3,14 +3,93 @@
 """
 Demonstration of Graph and BokehGraph functionality.
 """
-
+import random
 from sys import argv
+from graph import Graph
+from draw import BokehGraph
+
+def createDefaultGraph():
+    graph=Graph()
+    graph.add_vertex('0')
+    graph.add_vertex('1')
+    graph.add_vertex('2')
+    graph.add_vertex('3')
+    graph.add_vertex('4')
+    graph.add_vertex('5')
+    graph.add_vertex('6')
+    graph.add_vertex('7')
+    graph.add_vertex('8')
+    graph.add_undirected_edge('0', '1')
+    graph.add_undirected_edge('0', '3')
+    graph.add_directed_edge('1', '2')
+    graph.add_directed_edge('1', '4')
+    graph.add_directed_edge('2', '5')
+    graph.add_directed_edge('2', '6')
+    graph.breadth_first_search('0')
+    graph.depth_first_search('0')
+    graph.dfs_search_value('0','1')
+    #graph.dfs_search_value('0','11')
+    #graph.add_directed_edge('7','4')
+    #graph.add_directed_edge('7','4')
+    g1 = BokehGraph(graph)
+    g1.show()
+
```

Comment:
I am getting three graphs everytime I run graph_demo.py. I think one of them is from your draw.py file. The other two are from this file - one is the default graph, and the other is the random graph. I think this is because you run the show() method for each one.